src/Jobs/OneTimeOperationRollbackJob.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+namespace TimoKoerber\LaravelOneTimeOperations\Jobs;
+
+use Illuminate\Bus\Queueable;
+use Illuminate\Contracts\Queue\ShouldQueue;
+use Illuminate\Foundation\Bus\Dispatchable;
+use Illuminate\Queue\InteractsWithQueue;
+use Illuminate\Queue\SerializesModels;
+use Throwable;
+use TimoKoerber\LaravelOneTimeOperations\Models\FailedOperation;
+use TimoKoerber\LaravelOneTimeOperations\Models\Operation;
+use TimoKoerber\LaravelOneTimeOperations\OneTimeOperationManager;
+
+class OneTimeOperationRollbackJob implements ShouldQueue
+{
+    use Dispatchable, InteractsWithQueue, Queueable, SerializesModels;
+
+    public string $operationName;
+
+    public function __construct(string $operationName)
+    {
+        $this->operationName = $operationName;
+    }
+
+    /**
+     * Execute the rollback.
+     *
+     * @throws Throwable
+     */
+    public function handle(): void
+    {
+        $operationClassInstance = OneTimeOperationManager::getClassObjectByName($this->operationName);
+        try {
+            $operationClassInstance->rollback();
+        } catch (Throwable $th) {
+            $operation = Operation::where('name', $this->operationName)->first();
+
+            if ($operation) {
+                FailedOperation::create([
```

Comment:
The same model is used yo log failed operations while processing so we need another way to differentiate between processing and rollback 
- Create New Model For Failed Operation rollback
- Create new column contains type of failure processing or rolling back